tests/e2e/unified_test.go

```diff
@@ -241,26 +241,31 @@ func testFixturesInSeries(ctx context.Context, t *testing.T, testPause bool, can
 					}
 
 					for _, event := range events.HTTPEvents {
-						id := ""
+						operationID := ""
 						body := event.Response.ParseBody()
 						val, ok := body["name"]
 						if ok {
 							s := val.(string)
 							// operation name format: operations/{operationId}
 							if strings.HasPrefix(s, "operations/") {
-								id = strings.TrimPrefix(s, "operations/")
+								operationID = strings.TrimPrefix(s, "operations/")
 							}
 							// operation name format: {prefix}/operations/{operationId}
 							if ix := strings.Index(s, "/operations/"); ix != -1 {
-								id = strings.TrimPrefix(s[ix:], "/operations/")
+								operationID = strings.TrimPrefix(s[ix:], "/operations/")
 							}
 							// operation name format: operation-{operationId}
 							if strings.HasPrefix(s, "operation") {
-								id = s
+								operationID = s
+							}
+							// name format: accessPolicies/accessPolicyID}/accessLevels/{uniqueId}
```

Comment:
Nit: It doesn't matter, but it may be clearer to put this into a separate method / loop - even if we have to call ParseBody again and it's much less efficient, that way we keep this one to be about operations.